### PR TITLE
Updates ERT gear and adds holo stretcher

### PR DESCRIPTION
**Pull request by warior4356**
## What Does This PR Do

- Moves all ERT firearms from their right hand to their left.
- Moves T4 charges and mindshields for ERT from lockboxes to cardboard boxes.
- Gives the Red and Gamma ERT medics a bottle of painkillers.
- Makes ERT PDAs detonate immune and gives them the same functionality as the captain.
- Adds a holo stretcher, a roller bed that fits in a pocket. All levels of medical ERT start with one in lieu of a roller bed.
- Science can make a holo stretcher with biotech 5, magnets 4, and power systems 4. It takes 1000 metal, 500 silver, 500 glass, and 200 diamond.
- The surgery success chance of a roller bed was taken from 70% to 90%.

## Why It's Good For The Game

- A large number of friendly fire incidents are caused by the guns being selected when the ERT spawns in.
- It's very frustrating that ERT starts with items in their backpacks that don't fit in a backpack, and it's not like the lockbox is going to keep anyone out since killing the ERT will give you an ID that will open the lockbox.
- Doing surgery without painkillers or anesthetics is very difficult.
- The fact that ERT has PDAs on par with civilians is just sad, and it provides them a useful set of tools.
- Like with the lockboxes, this exists to remove an item from ERT that won't fit back in their backpacks.
- The item seemed useful and had a very lovely sprite courtesy of Stokes52, so I wanted the item to see more use. It was balanced to be cheap to make but requires science to have gotten materials from mining before it can be made. 
- Currently, a medical bed has a lower surgery success chance than a kitchen table, as it shares a 70% success rate with all other beds. This keeps it inferior to operation tables but makes them the next best option for operating. Further, it makes it practical, even if not ideal for ERT to do field surgery.

## Images of changes
image

## Changelog
:cl:
add: Added holo stretcher to ERT loadouts and science
add: Added a bottle of painkillers to Red and Gamma ERTs
add: Added all tools and detonate immunity to ERT PDAs
tweak: Moved ERT T4 and mindshields from lockboxes to cardboard
tweak: Moved ERT guns from their right hard to their left
tweak: Increased surgery success rate for roller beds to 90%.
imageadd: Added sprites for holo stretcher

/:cl:

### CODE REVIEW

**Comment by LiberatedWaveMan** [comment]
I'm no coder so I can't attest to the code or anything, but from the changelog and images provided? This looks like some great work.

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm [hunk 1]
```diff
@@ -64,9 +64,12 @@
 	name = "roller bed"
 	icon = 'icons/obj/rollerbed.dmi'
 	icon_state = "down"
+	var/up_state = "up"
+	var/down_state = "down"
```
Comment (SteelSlayer) [inline code/game/objects/structures/stool_bed_chair_nest/bed.dm:68]: Formatting nitpick: These should go down with `var/folded`, So we have all the new subtype variables together:
```dm
/obj/structure/bed/roller
	name = "roller bed"
	//etc 
	var/up_state
	var/down_state
	var/folded
```
Also autodoc would be nice for the vars.

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm [hunk 2]
```diff
@@ -114,6 +125,17 @@
...
 
+/obj/item/roller/holo
+	name = "holo stretcher"
+	desc = "A retracted hardlight stretcher that can be carried around."
+	icon_state = "holo_retracted"
+	w_class = WEIGHT_CLASS_SMALL
+	origin_tech = "magnets=3;biotech=4;powerstorage=3"
+	extended = /obj/structure/bed/roller/holo
+
+/obj/item/roller/holo/attackby(obj/item/W as obj, mob/user as mob, params)
```
Comment (SteelSlayer) [inline code/game/objects/structures/stool_bed_chair_nest/bed.dm:136]: This is legacy code.
```suggestion
/obj/item/roller/holo/attackby(obj/item/W, mob/user, params)
```

File: code/modules/pda/pdas.dm [hunk 3]
```diff
@@ -90,6 +90,8 @@
 	icon_state = "pda-h"
 
 /obj/item/pda/heads/ert
+	default_cartridge = /obj/item/cartridge/centcom
+	detonate = 0
```
Comment (SteelSlayer) [inline code/modules/pda/pdas.dm:94]:
```suggestion
	detonate = FALSE
```

**Comment by Dumbdumn5** [comment]
Copy-pasting from my initial impressions:  ERT PR + HS, I love the idea of the holo-stretcher and the removal of the lockbox bit due to common sense, painkillers for field surgery in an emergency situation is fucking cool and having an actual toolset built into the ERT PDA is fucking nice. :+1:

**Review by Kyep — APPROVED** [review]

**Review by SabreML — APPROVED** [review]

**Comment by SabreML** [comment]
The runtimes in the CI weren't your fault in this case (#16441). Closing and re-opening the PR will fix them.

**Comment by warior4356** [comment]
As sabre requested, closing and reopening

**Pull request closed without merging**